src/tests/test_common/hs_repl_test_common.hpp

```diff
@@ -56,7 +54,7 @@ namespace test_common {
 VENUM(ipc_packet_op_t, uint32_t, WAKE_UP = 0, CLEAN_EXIT = 1, UNCLEAN_EXIT = 2, PEER_GOING_DOWN = 3);
 ENUM(repl_test_phase_t, uint32_t, REGISTER, MEMBER_START, TEST_RUN, VALIDATE, CLEANUP);
 
-class HSReplTestHelper {
+class HSReplTestHelper : public HSTestHelper {
```

Comment:
Made ReplTestHelper a derived class from HSTestHelper, which is the right and natural thing to do. Earlier that was not possible because of static methods in HSTestHelper.